### PR TITLE
Prepare readme for upcoming beta

### PR DESCRIPTION
**Pull request by PotatoQuality**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,8 +90,15 @@ Search the existing requests to see if your suggestion has already been submitte
...
 
-[Windows Guide!](http://www.htpcguides.com/install-plex-requests-net-windows-system-service/)  
-[Ubuntu Guide!](http://www.htpcguides.com/install-plex-requests-net-ubuntu-14-x/)
+Windows: Download the windows zip file above and run ombi.exe
+
+Linux: Download the linux zip file, run chmod +x Ombi to make the Ombi file an executable.
+
+Get the following error? 
+libunwind.so.8: cannot open shared object file
+Sometimes you need to install libwind8.
```
Comment (camjac251) [inline README.md]: You may need to install libwind8.

File: README.md [hunk 2]
```diff
@@ -14,46 +14,67 @@ ____
...
 * Pushbullet
 * Pushover
-* Slack
-* Discord
-* Weekly Recently Added email notification to all of your Plex Users!
+* Mattermost
+
+### The difference between Version 3 and 2
+
+Over the last 7 months, we focust on the main functions on Ombi.
```
Comment (camjac251) [inline README.md]: focust is mispelled. Focused

File: README.md [hunk 3]
```diff
@@ -14,46 +14,67 @@ ____
...
 * A landing page that will give you the availability of your Plex server and also add custom notification text to inform your users of downtime.
-* Allow your users to get notifications!
+* Allow your users to get custom notifications!
 * Secure authentication so you don't have to worry about those script kiddies
 * We check to see if the request is already in Plex, if it's already in Plex then why you requesting it?!
-* We have allowed the ability for a user to add a custom note on a request
 * It automatically updates the status of requests when they are available on Plex
 * Slick, responsive and mobile friendly UI
-
+* Ombi wil automatically update here self :)
```
Comment (camjac251) [inline README.md]: Ombi will automatically update itself :)

File: README.md [hunk 4]
```diff
@@ -14,46 +14,67 @@ ____
...
 * Easily manage your requests
 * User management system (supports plex.tv accounts and local accounts)
-* Sending newsletters
-* Fault Queue for requests (Buffer requests if Sonar/Couchpotato/SickRage is offline)
 * Allow your users to report issues and manage them separately
 * A landing page that will give you the availability of your Plex server and also add custom notification text to inform your users of downtime.
-* Allow your users to get notifications!
+* Allow your users to get custom notifications!
 * Secure authentication so you don't have to worry about those script kiddies
 * We check to see if the request is already in Plex, if it's already in Plex then why you requesting it?!
```
Comment (camjac251) [inline README.md]: * We check to see if the request is already on Plex, if it's already on Plex then why you requesting it?!

File: README.md [hunk 5]
```diff
@@ -14,46 +14,67 @@ ____
...
+| Download |[![Download](http://i.imgur.com/odToka3.png)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/DotNetCore/artifacts)            |      [![Download](http://i.imgur.com/odToka3.png)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/eap/artifacts)       |  [![Download](http://i.imgur.com/odToka3.png)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/dev/artifacts)            |
 # Features
-Here some of the features Ombi has:
-* All your users to Request Movies, TV Shows (Whole series, whole seaons or even single episodes!) and Albums
+Here some of the features Ombi V3 has:
+* Out of the box working on all OS Just download the zip and start Ombi!
+* Let users request Movies and TV Shows (Whole series, whole seaons or even single episodes!)
```
Comment (camjac251) [inline README.md]: * Lets users request Movies and TV Shows (whether it being the entire series, an entire season, or even single episodes.)

File: README.md [hunk 6]
```diff
@@ -14,46 +14,67 @@ ____
...
+| Download |[![Download](http://i.imgur.com/odToka3.png)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/DotNetCore/artifacts)            |      [![Download](http://i.imgur.com/odToka3.png)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/eap/artifacts)       |  [![Download](http://i.imgur.com/odToka3.png)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/dev/artifacts)            |
 # Features
-Here some of the features Ombi has:
-* All your users to Request Movies, TV Shows (Whole series, whole seaons or even single episodes!) and Albums
+Here some of the features Ombi V3 has:
+* Out of the box working on all OS Just download the zip and start Ombi!
```
Comment (camjac251) [inline README.md]: * Now working without crashes on Linux.

File: README.md [hunk 7]
```diff
@@ -14,46 +14,67 @@ ____
 [![Report a bug](http://i.imgur.com/xSpw482.png)](https://github.com/tidusjar/Ombi/issues/new) [![Feature request](http://i.imgur.com/mFO0OuX.png)](http://feathub.com/tidusjar/Ombi)
 
 
-| Service  | Master (Recommend)                     | Early Access                 | Dev                          | V3 (Not ready for use!)                   |
-|----------|:---------------------------:|:----------------------------:|:----------------------------:|:----------------------------:|
+| Service  | Master (Recommend Version 3.0)                     | Early Access                 | Dev                          |
```
Comment (camjac251) [inline README.md]: Master (Version 3.0 - Recommended)

File: README.md [hunk 8]
```diff
@@ -14,46 +14,67 @@ ____
...
 * Slick, responsive and mobile friendly UI
-
+* Ombi wil automatically update here self :)
+* Very fast system.
 
 ### Integration 
 We integrate with the following applications:
-* Plex server 1.2 (and higher)
-* Emby (beta)
+* Plex server
```
Comment (camjac251) [inline README.md]: Plex Media Server

File: README.md [hunk 9]
```diff
@@ -14,46 +14,67 @@ ____
...
 * Pushover
-* Slack
-* Discord
-* Weekly Recently Added email notification to all of your Plex Users!
+* Mattermost
+
+### The difference between Version 3 and 2
+
+Over the last 7 months, we focust on the main functions on Ombi.
+Make it better, faster and more stable.
```
Comment (camjac251) [inline README.md]: Over the last 7 months, we focused on the main functions on Ombi, a complete rewrite while making it better, faster and more stable.

Comment (camjac251) [reply README.md]: not sure if it's a rewrite, that part could be removed

File: README.md [hunk 10]
```diff
@@ -14,46 +14,67 @@ ____
...
+We did already a lot of work, but you will still miss some functions in the new version.
```
Comment (camjac251) [inline README.md]: We have already done most of the work, but some features will still be missing in this first version.

File: README.md [hunk 11]
```diff
@@ -14,46 +14,67 @@ ____
...
+We are planning to bring this features back in Version 3. You can find a list below with the feature difference.
```
Comment (camjac251) [inline README.md]: We are planning to bring back these features in Version 3 but for now you can find a list below with a comparison of features between v2 and v3.

File: README.md [hunk 12]
```diff
@@ -14,46 +14,67 @@ ____
...
+
+
+| Service  | Version 3 | Version 2 |
+|----------|:----------:|:----------:|
+| Supported online | Yes | No |
```
Comment (camjac251) [inline README.md]: What does supported online mean? Remote servers?

Remote Server Support

File: README.md [hunk 13]
```diff
@@ -14,46 +14,67 @@ ____
...
+| Emby & Plex support | Yes | Yes |
+| Mono needed | No | Yes |
```
Comment (camjac251) [inline README.md]: Mono dependency

File: README.md [hunk 14]
```diff
@@ -14,46 +14,67 @@ ____
...
+| Notifications support | Yes| Yes |
+| Landing page | Yes (brand new) | Yes |
+| Login page | Yes (brand new) | Yes |
+| Custom Logo in Ombi and notifications | Yes | No |
+| Change bulk roles in user management | Yes  | No |
```
Comment (camjac251) [inline README.md]: Does this mean bulk edits to roles of users? or classes?

File: README.md [hunk 15]
```diff
@@ -14,46 +14,67 @@ ____
...
-Here some of the features Ombi has:
-* All your users to Request Movies, TV Shows (Whole series, whole seaons or even single episodes!) and Albums
+Here some of the features Ombi V3 has:
+* Out of the box working on all OS Just download the zip and start Ombi!
+* Let users request Movies and TV Shows (Whole series, whole seaons or even single episodes!)
 * Easily manage your requests
 * User management system (supports plex.tv accounts and local accounts)
-* Sending newsletters
-* Fault Queue for requests (Buffer requests if Sonar/Couchpotato/SickRage is offline)
 * Allow your users to report issues and manage them separately
```
Comment (tidusjar) [inline README.md]: Id remove the part about issues, I've not even started that part

File: README.md [hunk 16]
```diff
@@ -14,46 +14,67 @@ ____
...
+| Download |[![Download](http://i.imgur.com/odToka3.png)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/DotNetCore/artifacts)            |      [![Download](http://i.imgur.com/odToka3.png)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/eap/artifacts)       |  [![Download](http://i.imgur.com/odToka3.png)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/dev/artifacts)            |
 # Features
-Here some of the features Ombi has:
-* All your users to Request Movies, TV Shows (Whole series, whole seaons or even single episodes!) and Albums
+Here some of the features Ombi V3 has:
+* Out of the box working on all OS Just download the zip and start Ombi!
+* Let users request Movies and TV Shows (Whole series, whole seaons or even single episodes!)
 * Easily manage your requests
 * User management system (supports plex.tv accounts and local accounts)
```
Comment (tidusjar) [inline README.md]: (supports Plex.tv, Emby and local accounts)

File: README.md [hunk 17]
```diff
@@ -14,46 +14,67 @@ ____
...
-* Sending newsletters
-* Fault Queue for requests (Buffer requests if Sonar/Couchpotato/SickRage is offline)
 * Allow your users to report issues and manage them separately
 * A landing page that will give you the availability of your Plex server and also add custom notification text to inform your users of downtime.
```
Comment (tidusjar) [inline README.md]: availability of your Plex/**Emby** server

File: README.md [hunk 18]
```diff
@@ -14,46 +14,67 @@ ____
...
-* Allow your users to get notifications!
+* Allow your users to get custom notifications!
 * Secure authentication so you don't have to worry about those script kiddies
 * We check to see if the request is already in Plex, if it's already in Plex then why you requesting it?!
-* We have allowed the ability for a user to add a custom note on a request
 * It automatically updates the status of requests when they are available on Plex
```
Comment (tidusjar) [inline README.md]: when they are available on Plex/**Emby**

File: README.md (hunk 9 above)
Comment (tidusjar) [reply README.md]: It is a rewrite

File: README.md (hunk 12 above)
Comment (tidusjar) [reply README.md]: Yeah I'd remove the supported online and add 

| Service  | Version 3 | Version 2 |
|----------|:----------:|:----------:|
| Multiple Plex/Emby Servers | Yes | No |

File: README.md (hunk 14 above)
Comment (tidusjar) [reply README.md]: I'd remove this part for now as it's also not done yet

File: README.md [hunk 19]
```diff
@@ -14,46 +14,67 @@ ____
...
+| Landing page | Yes (brand new) | Yes |
+| Login page | Yes (brand new) | Yes |
+| Custom Logo in Ombi and notifications | Yes | No |
+| Change bulk roles in user management | Yes  | No |
+| Sending newsletters | Planned | Yes |
+| Send a Mass Email | Planned | Yes |
+| SickRage | Planned (not supported yet)| Yes |
+| CouchPotato | Planned| Yes |
+| Watcher | Planned| Yes |
+| Headphones | X (support dropped) | Yes |
```
Comment (tidusjar) [inline README.md]: Can we change the **X** to **No** same as the rest?

**Comment by PotatoQuality** [comment]
I'm going to finish this tomorrow.

File: README.md (hunk 7 above)
Comment (PotatoQuality) [reply README.md]: Done

File: README.md (hunk 6 above)
Comment (PotatoQuality) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (PotatoQuality) [reply README.md]: Done

File: README.md (hunk 16 above)
Comment (PotatoQuality) [reply README.md]: Done

File: README.md (hunk 15 above)
Comment (PotatoQuality) [reply README.md]: Done

File: README.md (hunk 17 above)
Comment (PotatoQuality) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (PotatoQuality) [reply README.md]: Done

File: README.md (hunk 18 above)
Comment (PotatoQuality) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (PotatoQuality) [reply README.md]: Done

File: README.md (hunk 8 above)
Comment (PotatoQuality) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (PotatoQuality) [reply README.md]: Done

File: README.md (hunk 9 above)
Comment (PotatoQuality) [reply README.md]: Done

File: README.md (hunk 10 above)
Comment (PotatoQuality) [reply README.md]: Done

File: README.md (hunk 11 above)
Comment (PotatoQuality) [reply README.md]: Done

File: README.md (hunk 12 above)
Comment (PotatoQuality) [reply README.md]: Done

File: README.md (hunk 13 above)
Comment (PotatoQuality) [reply README.md]: Done

File: README.md (hunk 14 above)
Comment (PotatoQuality) [reply README.md]: Done

File: README.md (hunk 19 above)
Comment (PotatoQuality) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (PotatoQuality) [reply README.md]: done

**Comment by PotatoQuality** [comment]
Thanks guys for review.

File: README.md [hunk 20]
```diff
@@ -14,46 +14,65 @@ ____
...
+| Service  | Master (Version 2.0 - Recommended)                | Open Beta     (Version 3.0)          |
+|----------|:---------------------------:|:----------------------------:|
+| AppVeyor | [![Build status](https://ci.appveyor.com/api/projects/status/hgj8j6lcea7j0yhn/branch/master?svg=true)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/master) | [![Build status](https://ci.appveyor.com/api/projects/status/hgj8j6lcea7j0yhn/branch/DotNetCore?svg=true)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/DotNetCore) | 
+| Download |[![Download](http://i.imgur.com/odToka3.png)](https://github.com/tidusjar/Ombi/releases)            |      [![Download](http://i.imgur.com/odToka3.png)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/DotNetCore/artifacts)       | 
 # Features
-Here some of the features Ombi has:
```
Comment (camjac251) [inline README.md]: Here's

File: README.md [hunk 21]
```diff
@@ -14,46 +14,65 @@ ____
...
-* All your users to Request Movies, TV Shows (Whole series, whole seaons or even single episodes!) and Albums
+Here some of the features Ombi V3 has:
```
Comment (camjac251) [inline README.md]: Here are some of the features Ombi V3 has:

File: README.md [hunk 22]
```diff
@@ -14,46 +14,65 @@ ____
...
 * Easily manage your requests
-* User management system (supports plex.tv accounts and local accounts)
-* Sending newsletters
-* Fault Queue for requests (Buffer requests if Sonar/Couchpotato/SickRage is offline)
-* Allow your users to report issues and manage them separately
-* A landing page that will give you the availability of your Plex server and also add custom notification text to inform your users of downtime.
-* Allow your users to get notifications!
+* User management system (supports plex.tv, Emby and local accounts)
+* A landing page that will give you the availability of your Plex/Emby server and also add custom notification text to inform your users of downtime.
+* Allow your users to get custom notifications!
```
Comment (camjac251) [inline README.md]: Allows your users to get custom notifications!

File: README.md [hunk 23]
```diff
@@ -14,46 +14,65 @@ ____
...
 * Secure authentication so you don't have to worry about those script kiddies
-* We check to see if the request is already in Plex, if it's already in Plex then why you requesting it?!
-* We have allowed the ability for a user to add a custom note on a request
-* It automatically updates the status of requests when they are available on Plex
+* We check to see if the request is already on Plex, if it's already on Plex then why you requesting it?!
```
Comment (camjac251) [inline README.md]: Will show if the request is already on plex or even if it's already monitored.

File: README.md [hunk 24]
```diff
@@ -14,46 +14,65 @@ ____
...
+* It automatically updates the status of requests when they are available on Plex/Emby
```
Comment (camjac251) [inline README.md]: Automatically updates the status of requests when they are available on Plex/Emby

File: README.md [hunk 25]
```diff
@@ -14,46 +14,65 @@ ____
...
 * Pushover
-* Slack
-* Discord
-* Weekly Recently Added email notification to all of your Plex Users!
+* Mattermost
+
+### The difference between Version 3 and 2
+
+Over the last 7 months, we focused on the main functions on Ombi, a complete rewrite while making it better, faster and more stable.
+We have already done most of the work, but some features will still be missing in this first version.
```
Comment (camjac251) [inline README.md]: We have already done most of the work, but some features are still be missing in this first version.

File: README.md [hunk 26]
```diff
@@ -69,8 +88,15 @@ Search the existing requests to see if your suggestion has already been submitte
...
-[Windows Guide!](http://www.htpcguides.com/install-plex-requests-net-windows-system-service/)  
-[Ubuntu Guide!](http://www.htpcguides.com/install-plex-requests-net-ubuntu-14-x/)
+Windows: Download the windows zip file above and run ombi.exe
+
+Linux: Download the linux zip file, run chmod +x Ombi to make the Ombi file an executable.
+
+Get the following error? 
+libunwind.so.8: cannot open shared object file
+You may need to install libwind8.
+Pleas run the following command:
```
Comment (camjac251) [inline README.md]: This line can be removed.

**Pull request merged**